packages/front-end/components/Layout/SidebarLink.tsx

```diff
@@ -50,10 +50,6 @@ const SidebarLink: FC<SidebarLinkProps> = (props) => {
     }
   }, [selected]);
 
-  if (props.feature && !growthbook.isOn(props.feature)) {
-    return null;
-  }
```

Comment:
Don't delete this.  We may want to use feature flags for left-nav items in the future so it's good to keep that capability.